src/devtools/client/webconsole/constants.js

```diff
@@ -32,16 +32,12 @@ const actionTypes = {
   MESSAGES_CLEAR_EVALUATIONS: "MESSAGES_CLEAR_EVALUATIONS",
   MESSAGES_CLEAR_EVALUATION: "MESSAGES_CLEAR_EVALUATION",
   MESSAGES_CLEAR_LOGPOINT: "MESSAGES_CLEAR_LOGPOINT",
-  NETWORK_MESSAGE_UPDATE: "NETWORK_MESSAGE_UPDATE",
-  NETWORK_UPDATE_REQUEST: "NETWORK_UPDATE_REQUEST",
   PERSIST_TOGGLE: "PERSIST_TOGGLE",
   PRIVATE_MESSAGES_CLEAR: "PRIVATE_MESSAGES_CLEAR",
   REMOVE_NOTIFICATION: "REMOVE_NOTIFICATION",
   FRONTS_TO_RELEASE_CLEAR: "FRONTS_TO_RELEASE_CLEAR",
   REVERSE_SEARCH_INPUT_TOGGLE: "REVERSE_SEARCH_INPUT_TOGGLE",
-  SELECT_NETWORK_MESSAGE_TAB: "SELECT_NETWORK_MESSAGE_TAB",
   SHOW_OBJECT_IN_SIDEBAR: "SHOW_OBJECT_IN_SIDEBAR",
```

Comment:
Did you mean to take out all references to `SHOW_OBJECT_IN_SIDEBAR`?

Reply:
we should take that out in another PR.